### DISCUSSION TITLE
Thesis support

### DISCUSSION THREAD

**Discussion by awelormro** [jgm/pandoc#10033] [General]
Hello to all who is involved in this project.

This report is to ask to all the team to ask you if you can help me in a sort of problem I have: I am writing a thesis about pandoc and how it can be used to help college students against technology gap and use FOSS as an alternative to combat obsolescence, but in my university are not huge fans of website info and they asked me if I can interview people involved in the project, can you help me with it? It will not be as technical as you think, I'm a renewable energy engineer but in this case, pandoc is an example of sustainable software in a very good way. 

Also consider this as an appreciation post for all the effort all of you have put into the developing of pandoc. Thank you for helping me to find an open source alternative that can help people in my community and helping me to find a way to fuse sustainiability and software. If you cannot help me, won't be a problem, but it will be awesome if you can help me with this :)

TYIA

**Comment by jgm** [comment]
You can write to me directly.  My email is in pandoc.cabal.

**Reply by awelormro** [reply]
Thanks!

**Comment by tarleb** [comment]
If you need an additional interviewee than please feel free to ping me via mail.

**Reply by awelormro** [reply]
Sure I will need, is the mail in your profile aviable to send an email there? And thank you for collaborate

**Reply by tarleb** [reply]
Yes, that address should work. :+1:
